src/irep2/irep2_type.cpp

```diff
@@ -175,7 +175,7 @@ unsigned int vector_type2t::get_width() const
 
 unsigned int pointer_type2t::get_width() const
 {
-  return config.ansi_c.pointer_width;
+  return config.ansi_c.pointer_width(); /* CHERI-TODO */
```

Comment:
What is missing here?

Reply:
I've clarified the TODO. We basically need to take into account whether this is a pointer or a capability (i.e. `can_carry_provenance`).